File: Searching_Algorithms/Linear_Search/Linear_search.py
#takes the input of the size of the array
print("Enter the size of the array")
n = int(input())

#reads the elements in the array
print("Enter the array elements seperated by space")
arr = list(map(int, input().split()))

#take the input for the search element
print("Enter the element to be searched")
search = int(input())

flag = 0
#searches and prints the result
for i in range(n):
    if search == arr[i]:
        flag = 1
        print("Element found at the index ",(i+1))
        break

if flag == 0:
    print("Element not found !")

File: Searching_Algorithms/Linear_Search/linearsearch.c
#include<stdio.h>
int main()
{
    int n,a[100];
    int i,c=1,num;
    printf("ENter number of elements in array\n");
    scanf("%d",&n);
    for(i=0;i<n;i++)
        scanf("%d",&a[i]);
    printf("Enter number to be searched\n");
    scanf("%d",&num);
    for(i=0;i<n;i++)
        if(a[i]==num)
            {
            c=0;
            printf("Number found at position %d\n",i+1);
            break;
            }
    if(c==1)printf("Number not found\n");
    return 0;

}
